modules/snooker/src/main/ts/ephox/snooker/resize/Bars.ts

```diff
@@ -12,12 +12,31 @@ const resizeRowBar = Styles.resolve('resizer-rows');
 const resizeColBar = Styles.resolve('resizer-cols');
 const BAR_THICKNESS = 7;
 
+const resizableRows = (warehouse: Warehouse, isResizable: (elm: SugarElement<Element>) => boolean): number[] =>
+  Arr.bind(warehouse.all, (row, i) => isResizable(row.element) ? [ i ] : []);
+
+const resizableColumns = (warehouse: Warehouse, isResizable: (elm: SugarElement<Element>) => boolean): number[] => {
+  const resizableCols: number[] = [];
+  // Check col elements and see if they are resizable
+  Arr.range(warehouse.grid.columns, (index) => {
+    // With use of forall, index will be included if col doesn't exist meaning the column cells will be checked below
+    if (Warehouse.getColumnAt(warehouse, index).map((col) => col.element).forall((col) => isResizable(col))) {
```

Comment:
This is a rather long if statement? Not entirely fussed either way, but might be better in a variable or split up somehow?